src/js/components/modals/InstallPackageModal.js

```diff
@@ -532,6 +553,11 @@ class InstallPackageModal extends mixin(InternalStorageMixin, TabsMixin, StoreMi
     let {isLoading} = this.internalStorage_get();
     let cosmosPackage = CosmosPackagesStore.getPackageDetails();
 
+    // CLI only package
+    if (cosmosPackage && !cosmosPackage.getMetadata()) {
```

Comment:
I'd suggest creating a method on the `UniversePackage` struct called `isCLIOnly` or something like that.